src/dscom.build/ReadMe.md

```diff
@@ -0,0 +1,83 @@
+# dscom.build

```

Comment:
I would see this docu in the main README.md

Reply:
This should be on the same level as the Command Line Client and Library sections of the ReadMe.md

Reply:
Done in 04afc4b00309e1031b8019a9c1223c8eaddb37be